K8s/rbac.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: micro-registry
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - pods
+  verbs:
+  - list
+  - patch
+  - watch
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: micro-registry
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: micro-registry
+subjects:
```

Comment:
I think its not good choice to provide additional privileges to ```default``` service account.
A good choice would be to create a namespace , add a service account and give access to that service account.
 Also same Service account need to be get used for add deployment of application trying to access kubernetes APIs by ```github.com/micro/go-plugins/registry/kubernetes/v2```

Reply:
Will implement.